packages/icon/stories/Icon.stories.tsx

```diff
@@ -42,16 +40,50 @@ const shapes = {
 
 export default {
   title: "Graphic Elements/Icon",
-  decorators: [withKnobs],
-  component: Icon
-};
+  component: Icon,
+  argTypes: {
+    color: {
+      options: colors,
+      control: {
+        type: "select"
+      }
+    },
+    shape: {
```

Comment:
should this be predefined? see screenshot 
<img width="1579" alt="Screen Shot 2021-12-07 at 11 22 20 AM" src="https://user-images.githubusercontent.com/815936/145076504-823d3c96-c9a2-4e02-8034-f46c46a4c886.png">

Reply:
also, should ariaLabel and data-cy have fields?

Reply:
I have set the data-cy to not have control as I felt it was a bit noisy, also didn't seem like the best use for the controls when you're not seeing physical or behavioral changes with it that attribute. Thoughts? Happy to update all the stories that have the data-cy control disabled to enabled.

Reply:
Also TIL you can set a defaultValue for the argTypes. That should help with the first issue.